neos/neos-ui
# TASK: rewrite positional array sorter to behave like the php implementation

**What I did**
Adjust the ordering in the positional-array-sorter to behave more like the implementation in neos/utility-arrays (#2379): `before` and `after` positions may now be weighted, `start` and `after` positions will now be sorted appropriately according to descending weight. Positions like `after start` are no longer matched as `start` key.

**How I did it**
Mending the existing implementation did not prove easy as features like weighted `before` or `after` positions were not implemented, so I implemented the function anew.

**How to verify it**
I developed against the test cases (mostly ported from the php version). As a lot of parts in the UI use this functionality, I can't really tell how to best verify this in the actual UI.
